tests/tensor/test_basic.py

```diff
@@ -1783,15 +1780,21 @@ def test_broadcastable_flags_many_dims_and_inputs(self):
         c = TensorType(dtype=self.floatX, shape=(1, None, None, None, None, None))()
         d = TensorType(dtype=self.floatX, shape=(1, None, 1, 1, None, 1))()
         e = TensorType(dtype=self.floatX, shape=(1, None, 1, None, None, 1))()
+
         f = self.join_op(0, a, b, c, d, e)
         fb = tuple(s == 1 for s in f.type.shape)
-        assert not fb[0] and fb[1] and fb[2] and fb[3] and not fb[4] and fb[5]
+        assert f.type.shape == (5, 1, 1, 1, None, 1)
+        assert fb == (False, True, True, True, False, True)
```

Comment:
I kept the broadcastability assers even though this information is redundant from the current interpretation of `shape == 1`.
If in the future we re-introduce a `broadcastable` attribute, this should help updating the test.